presidio-analyzer/presidio_analyzer/predefined_recognizers/__init__.py

```diff
@@ -96,6 +97,7 @@
     "ItVatCodeRecognizer",
     "ItIdentityCardRecognizer",
     "ItPassportRecognizer",
+    "InGstinRecognizer",
```

Comment:
please also add to the recognizers yaml: https://github.com/microsoft/presidio/blob/main/presidio-analyzer/presidio_analyzer/conf/default_recognizers.yaml 
as disabled